.github/labeler.yml

```diff
@@ -21,34 +21,34 @@ CORE:
 Script:
 - changed-files:
     - any-glob-to-any-file: 
-      - src/server/scripts/**/*.cpp
-      - src/server/scripts/**/*.h
+      - 'src/server/scripts/**/*.cpp'
+      - 'src/server/scripts/**/*.h'
 
 UnitTests:
 - changed-files:
-    - any-glob-to-any-file: src/test/**/*
+    - any-glob-to-any-file: 'src/test/**/*'
 
 Documentation:
 - changed-files:
-    - any-glob-to-any-file: ./*.md
+    - any-glob-to-any-file: './*.md'
```

Comment:
this will probably solve it. I missed it

Reply:
or even better we change it to */*.md

Reply:
In the official examples all paths had ticks so I added them to all paths.
Also paths starting with "./" are supported as far as I can tell, so maybe we keep it like this for now?